pyodide-build/pyodide_build/config.py

```diff
@@ -151,6 +188,15 @@ def to_env(self) -> dict[str, str]:
 
 BUILD_VAR_TO_KEY = {v: k for k, v in BUILD_KEY_TO_VAR.items()}
 
+# Configuration keys that can be overridden by the user.
+# TODO: distinguish between variables that are overridable by the user and those that are not.
+OVERRIDABLE_BUILD_KEYS = {
+    "side_module_cflags",
```

Comment:
```suggestion
    "side_module_cflags",
    "side_module_cxxflags",
```

Reply:
Let's also remove "side_module" prefix as discussed in https://github.com/pyodide/pyodide/issues/4799#issuecomment-2151643233